go/oasis-test-runner/scenario/e2e/gas_fees_staking.go

```diff
@@ -63,9 +64,32 @@ func (sc *gasFeesImpl) Fixture() (*oasis.NetworkFixture, error) {
 
 	return &oasis.NetworkFixture{
 		Network: oasis.NetworkCfg{
-			NodeBinary:              f.Network.NodeBinary,
-			EpochtimeMock:           true,
-			StakingGenesis:          "tests/fixture-data/gas-fees/staking-genesis.json",
+			NodeBinary:    f.Network.NodeBinary,
+			EpochtimeMock: true,
+			StakingGenesis: &staking.Genesis{
+				Parameters: staking.ConsensusParameters{
+					DebondingInterval: 1,
+					GasCosts: transaction.Costs{
+						"transfer":       10,
```

Comment:
This should probably use the gas op constants defined in the staking api, e.g. this one would be `staking.GasOpTransfer`. Also elsewhere.